hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/recon/ReconConfigKeys.java

```diff
@@ -36,6 +36,12 @@ private ReconConfigKeys() {
       "ozone.recon.datanode.address";
   public static final String OZONE_RECON_ADDRESS_KEY =
       "ozone.recon.address";
+  public static final String OZONE_SOLR_ADDRESS_KEY =
+      "solr.address";
```

Comment:
configuration related to solr can be removed, solr.address and ozone.recon.solr.timezone